content/webapp/services/prismic/transformers/events.ts

```diff
@@ -34,30 +39,32 @@ export function transformEvent(
   const scheduleLength = isDocumentLink(data.schedule.map(s => s.event)[0])
     ? data.schedule.length
     : 0;
-  const genericFields = parseGenericFields(document);
+  const genericFields = transformGenericFields(document);
   const eventSchedule =
     scheduleQuery && scheduleQuery.results
       ? scheduleQuery.results.map(doc => transformEvent(doc))
       : [];
-  const interpretations = data.interpretations
+  const interpretations: Interpretation[] = data.interpretations
     .map(interpretation =>
       link(interpretation.interpretationType)
         ? {
             interpretationType: {
+              id: interpretation.interpretationType.id,
```

Comment:
Once again TypeScript comes to our rescue, by highlighting fields like this that were just missing.